docker/project.clj

```diff
@@ -33,16 +33,23 @@
                  [org.clojure/java.data "1.0.86"]
                  [potemkin "0.4.5"]
                  [reply "0.4.4" :exclusions [javax.servlet/servlet-api]]
-                 [zero.one/fxl "0.0.5"]]
+                 [zero.one/fxl "0.0.5"]
+                 [org.apache.arrow/arrow-memory-netty "2.0.0"]
+                 [org.apache.arrow/arrow-memory-core "2.0.0"]
+                 [org.apache.arrow/arrow-vector "2.0.0" :exclusions [commons-codec com.fasterxml.jackson.core/jackson-databind ]]
+                 ]
   :profiles
   {:provided {:dependencies ~spark-deps}
    :uberjar {:aot :all :dependencies ~spark-deps}
    :dev {:dependencies [[enlive "1.1.6"]
-                        [midje "1.9.9"]]
+                        [midje "1.9.9"]
+                        [techascent/tech.ml.dataset "5.00-alpha-19"]
+                        [criterium "0.4.6"]]
          :plugins [[lein-ancient "0.6.15"]
                    [lein-cloverage "1.2.1"]
                    [lein-midje "3.2.2"]
                    [lein-cljfmt "0.7.0"]]
+         :resource-paths ["test-resources"]
```

Comment:
Why do we need this? Do we need to version control `test-resources/logback.xml`?

Reply:
Bringing in a "test" dependency to tech.ml.dataset made a lot of DEBUG logging appear.
Probably one of the dependencies of TMD has a concrete logger as dependency.
Not sure, if there is a better way to solve this.